packages/next-rspack/README.md

```diff
@@ -1,7 +1,7 @@
 # next-rspack (EXPERIMENTAL)
 
 > [!WARNING]
-> This package is currently experimental and actively developed and supported in Next.js’ `canary` branch. To use this, you must be using a published canary build of Next.js.
+> This package is a community effort and is currently experimental. Help improve Next.js and Rspack by providing feedback at https://github.com/vercel/next.js/discussions/77800
```

Comment:
You could say that it is maintained by the Rspack team to clarify it's not just random community members